FrontEnd/src/Appointment/AppointmentPage.jsx

```diff
@@ -15,6 +17,22 @@ function AppointmentPage({ userData }) {
   const [upcomingAppointmentData, setUpcomingAppointmentData] = useState([]);
   const userID = userData?.userID;
 
+  const acceptedAppointment = async (vendorID, customerID, appointmentID) => {
```

Comment:
`acceptedAppointment` feels like it's going to return a list of appointments, instead of being a function. Maybe change to `acceptAppointment`.